### PR TITLE
added an element to display number of remaining enemy ships

### PR DESCRIPTION
**Pull request by yang30537**

### CODE REVIEW

**Comment by Gerasheo** [comment]
This looks good for the most part.
However, if the result is surrender (at the end of the game) instead of sunk, the count does not go down.
I think this can be accomplished by changing line 47 of game.js to 
`        if (attack.result === "SUNK" || attack.result === "SURRENDER") {`

File: src/main/java/views/ApplicationController/index.ftl.html [hunk 1]
```diff
@@ -20,6 +20,11 @@ <h1>The Battleship game</h1>
     <li><icon class="occupied icon">&#9724</icon> PLACED</li>
 </ul>
 
+<div class="count">
+    REMAINING
```
Comment (sellenth) [inline src/main/java/views/ApplicationController/index.ftl.html]: Div elements are usually used as generic content containers. If you want to have text within it you should use a more specific text tag like a header or paragraph.

File: src/main/java/assets/game.js [hunk 2]
```diff
@@ -39,6 +40,16 @@ function markHits(board, elementId, surrenderText) {
     });
 }
 
+function updateRemain(board) {
+    shipRemain = 3; 
+    board.attacks.forEach((attack) => {
+        if (attack.result === "SUNK") {
```
Comment (Gerasheo) [inline src/main/java/assets/game.js]:
```suggestion
        if (attack.result === "SUNK" || attack.result === "SURRENDER") {
```

**Comment by Gerasheo** [comment]
@sellenth I wrote mine so that it adds a different result type and won't interact with his counter.

**Pull request merged**